ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotBlockRetriever.java

```diff
@@ -148,6 +149,11 @@ private void handleContestedPivotBlock(final long contestedBlockNumber) {
       LOG.info("Received conflicting pivot blocks for {}.", contestedBlockNumber);
 
       final int retryCount = confirmationTasks.size();
+
+      if ((retryCount % SUSPICIOUS_NUMBER_OF_RETRIES) == 0) {
+        LOG.warn("Several attempts have been made without finding a pivot block");
```

Comment:
(optional) Might be worth adding some more specific details:
```suggestion
        LOG.warn("{} attempts have failed to find a fast sync pivot block", retryCount);
```

Reply:
Done